### ISSUE TITLE
Text in "Edit Classroom" page not aligning with page.

### ISSUE THREAD

**Issue by TeachTheWay** [ram8647/gcb-mobilecsp#22]
![defect_textonsectioncreationscreen](https://cloud.githubusercontent.com/assets/22352628/18723498/c58551a4-8006-11e6-8aeb-a25da3ca7d47.PNG)

**Comment by ram8647** [comment]
Yes, this is known bug.  Those messages are only supposed to be visible when the user hovers over the question marks.
